src/main/java/com/heycar/pairingsession/Listing.java

```diff
@@ -0,0 +1,91 @@
+package com.heycar.pairingsession;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Listing {
+
+    @Id
+    private String id;
+    @Column
+    private String dealerId;
```

Comment:
I would remove this column to not give hint. It's interesting to see how candidate makes decision
to create or not create entity for dealer